src/index.html

```diff
@@ -217,6 +217,10 @@
             saveNotes();
         })
 
+        editor.addEventListener('paste', (e) => {
+            console.log(e);
+        });
```

Comment:
Woops - this snuck in from some experimentation I was doing around being able to paste images from the clipboard. I can remove it if you like, or leave it as a useful stepping stone.

Reply:
Yeah, let's keep! :)